data/src/main/java/com/example/data/di/RepositoryModule.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.data.di
+
+import com.example.data.repository.GitHubRepositoryImpl
+import com.example.domain.repository.GitHubRepository
+import dagger.Binds
+import dagger.Module
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+
+@InstallIn(SingletonComponent::class)
+@Module
+abstract class RepositoryModule {
+    @Binds
```

Comment:
```suggestion
    @Binds
    @Singleton
```
여기서도 사실 싱글톤 어노테이션을 붙혀주는게 맞습니다!
한 번 이것도 찾아서 공부해보면 좋을 것 같아요! ><

Reply:
해놓고 저도 안썼네요 죄송합니다